app/footer.tsx

```diff
@@ -1,3 +1,170 @@
+        <div className="flex  flex-col items-center justify-center items px-10 ">
+          <Image src={logo} alt="" className=" h-[66px] w-[66px]" />
+          <div className="font-display text-[22px] pt-3 ">
+            National Institute of
+          </div>
+          <div className="font-display text-[22px]">
+            Technology, Kurukshetra
+          </div>
+          <div className="font-sans text-[16px] text-[#8B949E] pt-3">
+            WRXF+23G, NIT, Thanesar, Haryana
+          </div>
+          <div className="font-sans text-[16px] text-[#8B949E]">136119</div>
+
+          <div className="flex flex-row items-center justify-center pt-3">
+            <div className="pr-5">
+              <Image src={call} alt="" className="" />
+            </div>
+            <div className="pr-5">
+              <Image src={mail} alt="" className="" />
+            </div>
+            <div className="pr-5">
+              <Image src={lang} alt="" className="" />
+            </div>
+            <div className="pr-5">
+              <Image src={distance} alt="" className="" />
+            </div>
+          </div>
+        </div>
+
+        <div className="sm:items-center flex lg:items-center justify-center origin-center rotate-90 lg:rotate-0 ">
+          <Image src={footerimg} className="h-[362px] w-[74px] " alt="" />
+        </div>
+
+        <div className="flex lg:items-center justify-center lg:flex-row  flex-col flex-wrap sm:items-start ">
+          
+          <div className="lg:px-10  px-5 ">
+            <ul>
+              <li className="  font-black md:text-[24px] py-3 sm:text-[16px]">
+                Quick Links
+              </li>
+              <li className=" font-medium text-[#8B949E] md:text-[20px] py-1 sm:text-[12px]">
+                Telephone Directory
+              </li>
+              <li className=" font-medium text-[#8B949E] py-1 md:text-[20px] sm:text-[12px]">
+                Books and e-library
+              </li>
+              <li className=" font-medium text-[#8B949E] py-1 md:text-[20px] sm:text-[12px]">
+                Important Links
+              </li>
+              <li className=" font-medium text-[#8B949E] py-1 md:text-[20px] sm:text-[12px]">
+                Downloads
+              </li>
+
+              <li className=" font-medium text-[#8B949E] py-1 md:text-[20px] sm:text-[12px]">
+                RTI
+              </li>
+              <li className=" font-medium text-[#8B949E] py-1 md:text-[20px] sm:text-[12px]">
+                NIT KKR IT Infrastructure Usage Policy
+              </li>
+              <li className=" font-medium text-[#8B949E] py-1 md:text-[20px] sm:text-[12px]">
+                NIT KKR @NDL
+              </li>
+              <li className=" font-medium text-[#8B949E] py-1 md:text-[20px] sm:text-[12px]">
+                Telephone Directory
+              </li>
+            </ul>
+          </div>
```

Comment:
Make this into an object that is read from instead of manually initialising each list element